packages/cardano-services/src/Program/programs/providerServer.ts

```diff
@@ -189,36 +195,55 @@ const serviceMapFactory = (options: ServiceMapFactoryOptions) => {
     return sharedHandleProvider;
   };
 
-  return {
-    [ServiceNames.Asset]: withDbSyncProvider(async (dbPools, cardanoNode) => {
-      const ntfMetadataService = new DbSyncNftMetadataService({
-        db: dbPools.main,
-        logger,
-        metadataService: createDbSyncMetadataService(dbPools.main, logger)
-      });
-      const tokenMetadataService = args.tokenMetadataServerUrl?.startsWith('stub:')
-        ? new StubTokenMetadataService()
-        : new CardanoTokenRegistry({ logger }, args);
-      const assetProvider = new DbSyncAssetProvider(
-        {
-          cacheTTL: args.assetCacheTTL,
-          disableDbCache: args.disableDbCache,
-          paginationPageSizeLimit: Math.min(args.paginationPageSizeLimit! * 10, PAGINATION_PAGE_SIZE_LIMIT_ASSETS)
+  const getTypeormAssetProvider = withTypeOrmProvider('Handle', (connectionConfig$) => {
```

Comment:
```suggestion
  const getTypeormAssetProvider = withTypeOrmProvider('Asset', (connectionConfig$) => {
```

Reply:
`withTypeOrmProvider` receives [this](https://github.com/input-output-hk/cardano-js-sdk/blob/cfef7f30d1b932d7b30288c8c89c00fc8bd2aaa5/packages/cardano-services/src/Program/options/postgres.ts#L35) type, except I added `Asset` to ConnectionName

Reply:
We should add 'Asset' to that type union

Reply:
Fixed on 764cb98